README.md

```diff
@@ -0,0 +1,16 @@
+# Azure Event Wishlist
+
+Azure emits a variety of events to Azure Event Grid which allows you to extend and react to what's going on in your apps & infrastructure, but not everything is there yet.
+
+**Azure Event Wishlist** is a dashboard that gathers all events that would be great to have in Azure!
+
+## How can you help?
+
+- **Propose new event suggestions** by opening a new issue
+- **Vote for an event** that you'd like to be supported by giving a :+1: on the issue
```

Comment:
```suggestion
- **Vote for the events** you'd like to see supported by giving a :+1: on the issues
```

Reply:
Should the list have a third item, “**Discuss events** to provide valuable information to the Microsoft service teams”?